safe_landing_planner/src/nodes/waypoint_generator.cpp

```diff
@@ -66,13 +88,16 @@ void WaypointGenerator::updateSLPState() {
 
 SLPState WaypointGenerator::chooseNextState(SLPState currentState, usm::Transition transition) {
   prev_slp_state_ = currentState;
-  USM_TABLE(currentState, SLPState::GOTO,
-            USM_STATE(transition, SLPState::GOTO, USM_MAP(usm::Transition::NEXT1, SLPState::ALTITUDE_CHANGE);
-                      USM_MAP(usm::Transition::NEXT2, SLPState::LOITER));
-            USM_STATE(transition, SLPState::ALTITUDE_CHANGE, USM_MAP(usm::Transition::NEXT1, SLPState::LOITER));
-            USM_STATE(transition, SLPState::LOITER, USM_MAP(usm::Transition::NEXT1, SLPState::LAND);
-                      USM_MAP(usm::Transition::NEXT2, SLPState::GOTO));
-            USM_STATE(transition, SLPState::LAND, ));
+  USM_TABLE(
```

Comment:
clang-format is really ugly here. I think turn it off with `// clang-format off` (and on again below with `// clang-format on`) and then make this a bit nicer, so that it is easier to read. One USM_MAP on each line, etc